targets/posix/nanoCLR/main.cpp

```
@@ -0,0 +1,124 @@
+int main(int argc, char **argv)
+{
+    PrintBanner();
+
+    // ── Collect .pe paths from command line ──────────────────────────────────
+    // Accepts:  [--assemblies] file.pe file2.pe ...
+    // The --assemblies flag is accepted but not required (mirrors nanoclr CLI).
+    std::vector<std::string> peFiles;
+
+    for (int i = 1; i < argc; i++)
+    {
+        if (std::strcmp(argv[i], "--assemblies") == 0)
```

Clang (14.0.6)

[warning] statement should be inside braces

(readability-braces-around-statements)